.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```

Comment:
you should add `.vscode/` as a line in a top level `.gitignore` 

https://g.co/gemini/share/7b9dfbfd86ca